data/json/items/ranged.json

```diff
@@ -5113,5 +5113,33 @@
     "burst": 0,
     "clip_size": 0,
     "reload": 0
-  }
+  },
+  {
+  "id": "Abzats",
+  "type": "GUN",
+  "symbol": "(",
+  "color": "dark_gray",
+  "name": "Heavy Automatic Shotgun",
+  "description": "Modified from the massive M2 Browning, this heavy machine gun has been re-chambered and re-bored for shotgun shells, and completely redesigned to be wielded unmounted by one user.",
+  "price": 750000,
+  "material": ["hardsteel", "wood"],
+  "flags": ["MODE_BURST", "RELOAD_EJECT"], 
+  "skill": "shotgun",
+  "ammo": "shot",
+  "weight": 16500,
+  "volume": 18,
+  "bashing": 12,
+  "cutting": 0,
+  "to_hit": -1,
+  "dispersion": 150,
+  "sight_dispersion": 75,
+  "aim_speed": 5,
+  "recoil": 180,
+  "durability": 8,
```

Comment:
Looks way too durable for a homemade gun. I'd say durability shouldn't be more than 6-7 even considering it's not made totally from scratch.